src/petals/client/routing/sequence_manager.py

```diff
@@ -461,10 +461,23 @@ def get_request_metadata(self, protocol: str, *args, **kwargs) -> Optional[Dict[
         :param kwargs: additional request context, such as remote peer ID
         :returns: msgpack-serialized metadata dict that will be passed alongside a given request
         """
-        return dict(points=self.policy.get_points(protocol, *args, **kwargs), active_adapter=self.config.active_adapter)
+        return dict(
+            points=self.policy.get_points(protocol, *args, **kwargs),
+            active_adapter=self.config.active_adapter,
+            tensor_names=self.get_tensor_names(protocol),
+        )
 
     def shutdown(self):
         self._thread.shutdown()
+        
+    def get_tensor_names(self, protocol):
```

Comment:
I believe we were supposed to have those values in rpc_info; However, the values from rpc_info are de facto obsolete and it would take time to fix them.

Can you please add a todo to ensure that these values are consistent wit rpc_info?